kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -597,6 +599,20 @@ private static boolean loadFromKubeconfig(Config config, String context, String
 
     return false;
   }
+
+  private static Context setCurrentContext(String context, Config config, io.fabric8.kubernetes.api.model.Config kubeConfig) {
```

Comment:
Do you think we should maybe move this method to `KubeConfigUtils`

Reply:
I haven't look deeply but would we be able to resolve https://github.com/fabric8io/kubernetes-client/issues/1512 if we expose this as public?

Reply:
@rohanKanojia so the idea in #1512 is as far as I understand it is to be able to set the current context to the Config, right? The "problem" with that is that this would  imply an update to plenty of settings like the current cluster, namespace etc. Could be achieved by calling Config#autoConfigure behind the scene. I'd rather fix this in its very own issue.